<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productAdder</class>
 <widget class="QWidget" name="productAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>699</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>584</width>
    <height>699</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>584</width>
    <height>699</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создать заказ</string>
  </property>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-20</y>
     <width>541</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Соберите заказ из блюд и товаров, указанных ниже. Для того, чтобы добавить пункт в заказ,&lt;br/&gt;кликните на желаемое блюдо два раза. После выберите способ оплаты и доставки, &lt;br/&gt;время, к которому подготовить или доставить еду, а после подтвердите заказ.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creditCardPayButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Оплатить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPlacerButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>650</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="placerStatusLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>670</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="foodsToAdd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>521</width>
     <height>221</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>Наименование</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Цена</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Свойства</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ингредиенты</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="deleteElement">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить элемент</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentOrderLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ваш заказ:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="orderBuilder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Наименование</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Кол-во</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Цена</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="phoneLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>20</width>
     <height>371</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneInput">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string notr="true">+7-000-000-00-00</string>
   </property>
   <property name="text">
    <string>+7----</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="takeLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Способ получения:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="deliveryTake">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Доставка</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="attendanceTake">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Самовывоз</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="cityLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Город:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="citySelector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Красноярск</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Дивногорск</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сосновоборск</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="streetLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Улица:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="streetInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Академика Киренского</string>
   </property>
  </widget>
  <widget class="QLabel" name="houseLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дом/стр:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="houseInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>26Б</string>
   </property>
  </widget>
  <widget class="QLabel" name="aptLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кв/офис:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aptInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="paymentMethodLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>460</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Способ оплаты:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="cashPayment">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Наличные</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="cardPayment">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>500</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Карта</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="cardnumberLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>530</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер карты:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardInputFirst">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>550</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardInputSecond">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardInputThird">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>550</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardInputFourth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardMonth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="monthYearCardLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок действия:</string>
   </property>
  </widget>
  <widget class="QLabel" name="CVVLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>590</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CVV/CVC:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CVCInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>590</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="placeOrderButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить заказ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardYear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>550</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="monthYearCardLabel_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>550</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ММ/ГГ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deliveryCounter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="timeToDeliver">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>620</y>
     <width>194</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время готовности/доставки:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
